uber/templates/preregistration/preregbase.html

```diff
@@ -39,9 +39,7 @@
                 onClick: function () {
                     $('.group_fields').addClass('hide');
                     $('.non_group_fields').removeClass('hide');
-                    if ($.field('badge_type').val() == '{{ c.PSEUDO_GROUP_BADGE }}') {
-                        updateBadgeTypeHiddenInput('{{ c.ATTENDEE_BADGE }}');
-                    }
+                    updateBadgeTypeHiddenInput('{{ c.ATTENDEE_BADGE }}');
```

Comment:
Note for future reference: This commit probably breaks the BADGE_TYPE_PRICES badge types, which are custom badge types with prices associated with them. MAGFest doesn't use those, but let's not merge this into, e.g., MFF's fork without fixing it.